### PR TITLE
Fix wrong cxx_extern return value

### PR DESCRIPTION
**Pull request by yujincheng08**

### CODE REVIEW

File: native/src/base/files.hpp [hunk 1]
```diff
@@ -37,11 +37,11 @@ int mkdirs(const char *path, mode_t mode);
...
-void cp_afc(const char *src, const char *dest);
-void mv_path(const char *src, const char *dest);
-void link_path(const char *src, const char *dest);
-void clone_attr(const char *src, const char *dest);
-void fclone_attr(int src, int dest);
+bool cp_afc(const char *src, const char *dest);
+bool mv_path(const char *src, const char *dest);
+bool link_path(const char *src, const char *dest);
+bool clone_attr(const char *src, const char *dest);
+bool fclone_attr(int src, int dest);
```
Comment (yujincheng08) [inline native/src/base/files.hpp:44]: https://github.com/topjohnwu/Magisk/blob/07835a3e0e44ad398a82294eccfa93994ccfd9ff/native/src/base/cxx_extern.rs#L174

File: native/src/base/files.hpp [hunk 2]
```diff
@@ -37,11 +37,11 @@ int mkdirs(const char *path, mode_t mode);
...
 bool frm_rf(int dirfd);
-void cp_afc(const char *src, const char *dest);
-void mv_path(const char *src, const char *dest);
-void link_path(const char *src, const char *dest);
-void clone_attr(const char *src, const char *dest);
-void fclone_attr(int src, int dest);
+bool cp_afc(const char *src, const char *dest);
+bool mv_path(const char *src, const char *dest);
+bool link_path(const char *src, const char *dest);
+bool clone_attr(const char *src, const char *dest);
```
Comment (yujincheng08) [inline native/src/base/files.hpp:43]: https://github.com/topjohnwu/Magisk/blob/07835a3e0e44ad398a82294eccfa93994ccfd9ff/native/src/base/cxx_extern.rs#L158

File: native/src/base/files.hpp [hunk 3]
```diff
@@ -37,11 +37,11 @@ int mkdirs(const char *path, mode_t mode);
...
 bool rm_rf(const char *path);
 bool frm_rf(int dirfd);
-void cp_afc(const char *src, const char *dest);
-void mv_path(const char *src, const char *dest);
-void link_path(const char *src, const char *dest);
-void clone_attr(const char *src, const char *dest);
-void fclone_attr(int src, int dest);
+bool cp_afc(const char *src, const char *dest);
+bool mv_path(const char *src, const char *dest);
+bool link_path(const char *src, const char *dest);
```
Comment (yujincheng08) [inline native/src/base/files.hpp:42]: https://github.com/topjohnwu/Magisk/blob/07835a3e0e44ad398a82294eccfa93994ccfd9ff/native/src/base/cxx_extern.rs#L141

File: native/src/base/files.hpp [hunk 4]
```diff
@@ -37,11 +37,11 @@ int mkdirs(const char *path, mode_t mode);
 ssize_t canonical_path(const char * __restrict__ path, char * __restrict__ buf, size_t bufsiz);
 bool rm_rf(const char *path);
 bool frm_rf(int dirfd);
-void cp_afc(const char *src, const char *dest);
-void mv_path(const char *src, const char *dest);
-void link_path(const char *src, const char *dest);
-void clone_attr(const char *src, const char *dest);
-void fclone_attr(int src, int dest);
+bool cp_afc(const char *src, const char *dest);
+bool mv_path(const char *src, const char *dest);
```
Comment (yujincheng08) [inline native/src/base/files.hpp:41]: https://github.com/topjohnwu/Magisk/blob/07835a3e0e44ad398a82294eccfa93994ccfd9ff/native/src/base/cxx_extern.rs#L124

File: native/src/base/files.hpp [hunk 5]
```diff
@@ -37,11 +37,11 @@ int mkdirs(const char *path, mode_t mode);
 ssize_t canonical_path(const char * __restrict__ path, char * __restrict__ buf, size_t bufsiz);
 bool rm_rf(const char *path);
 bool frm_rf(int dirfd);
-void cp_afc(const char *src, const char *dest);
-void mv_path(const char *src, const char *dest);
-void link_path(const char *src, const char *dest);
-void clone_attr(const char *src, const char *dest);
-void fclone_attr(int src, int dest);
+bool cp_afc(const char *src, const char *dest);
```
Comment (yujincheng08) [inline native/src/base/files.hpp:40]: https://github.com/topjohnwu/Magisk/blob/07835a3e0e44ad398a82294eccfa93994ccfd9ff/native/src/base/cxx_extern.rs#L107

**Review by topjohnwu — APPROVED** [review]

**Pull request merged**